packages/manualpeering/manualpeering.go

```diff
@@ -342,18 +396,18 @@ func (m *Manager) changeNeighborStatus(neighbor *gossip.Neighbor, connStatus Con
 	kp.setConnStatus(connStatus)
 }
 
-func (m *Manager) connectionDirection(p *peer.Peer) (ConnectionDirection, error) {
-	localPK := m.local.PublicKey().String()
-	peerPK := p.PublicKey().String()
-	if localPK < peerPK {
+func (m *Manager) connectionDirection(peerPK ed25519.PublicKey) (ConnectionDirection, error) {
+	localPKStr := m.local.PublicKey().String()
+	peerPKStr := peerPK.String()
+	if localPKStr < peerPKStr {
```

Comment:
I'm not a big fan of using the string method to compare as this could have unwanted side effects if we change it. Maybe better to use `bytes.Compare(pk1.Bytes(), pk2.Bytes()`

Reply:
Yeah, you are right, let's go with bytes